packages/salesforcedx-vscode-core/package.json

```diff
@@ -326,27 +326,27 @@
         },
         {
           "command": "sfdx.force.source.retrieve.current.source.file",
-          "when": "false"
+          "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest'"
```

Comment:
For these commands that are available when `resourceLangId != 'forcesourcemanifest'`, you can also add `editorIsOpen` to the when clause to restrict the command from being available when no file is open in the editor. 

For the commands that require `resourceLangId == 'forcesourcemanifest'`, `editorIsOpen` isn't necessary.

Reply:
Good call @allileong I'll get this updated with the next PR.